ProQuest.js

```diff
@@ -499,6 +517,61 @@ function getItemType(types) {
 	return guessType;
 }
 
+function scrapeEbraryResults(refs) {
+	// Since we can't chase URLs, let's get what we can from the page
+	
+	for(var i = 0; i < refs.length; i++) {
+		var ref = refs[i];
+		var hiddenData = ZU.xpathText(ref.html, './span');
+		var visibleData = ZU.xpathText(ref.html.getElementsByClassName('results_list_copy'), './text()').trim().split(';').reverse();
+		
+		var item = new Zotero.Item("book");
+
+		var date = /\(([\w\d\s]+)\)/.exec(visibleData[0]);
+		var place = /([\w,\s]+)\(/.exec(visibleData[0]);
```

Comment:
This regex and the previous are looking at a string like:
Los Angeles, CA, USA (June 2006)

This regex is meant to match everything before the ( character, which will include letters, spaces, and commas.
The previous regex will match what's between ( and ). This will not have commas.